repo-hygiene/index.mjs

```
@@ -0,0 +1,844 @@
+/* eslint import/no-unresolved: [2, { ignore: ['@actions/'] }] -- This script run in GitHub CI */
+
+import * as core from '@actions/core'
+import * as actionsGithub from '@actions/github'
+import yaml from 'yaml'
```

ESLint

[error] Unable to resolve path to module 'yaml'.

(import/no-unresolved)